src/rrd_dump.c

```diff
@@ -569,6 +569,7 @@ int rrd_dump(
                           "[--no-header|-n]\n"
                           "[--daemon|-d address]\n"
                           "file.rrd [file.xml]", options.argv[0]);
+            if (opt_daemon) free(opt_daemon);
```

Comment:
I have seen all kinds of code style in the source ;-) but I'm happy to adapt. Let me know..